src/drivers/distance_sensor/tfmini/TFMINI.cpp

```diff
@@ -198,7 +200,7 @@ TFMINI::collect()
 
 		// parse buffer
 		for (int i = 0; i < ret; i++) {
-			tfmini_parse(readbuf[i], _linebuf, &_linebuf_index, &_parse_state, &distance_m);
+			ret_parse = tfmini_parse(readbuf[i], _linebuf, &_linebuf_index, &_parse_state, &distance_m);
```

Comment:
TODO: review if this is correct
We want to only publish on successful parse of new valid distance.